build.gradle

```diff
@@ -7,7 +7,9 @@ repositories {
 }
 
 dependencies {
+    implementation 'junit:junit:4.13.1'
```

Comment:
Any particular reason for this upgrade? not sure if this would affect the rest of the project

Reply:
No, idk why or when I change it. I will change it right away!

Reply:
Great, thanks